packages/react-client/src/trackManager.ts

```diff
@@ -80,9 +83,12 @@ export const useTrackManager = <PeerMetadata, TrackMetadata>({
     // see `getRemoteOrLocalTrackContext()` explanation
     setCurrentTrackId(media.track.id);
 
+    const trackMetadata = { type: getType(), paused: false };
```

Comment:
WDYT about having some type for such variables, so we can use it for easier refactoring in future?

Reply:
Refactored as @czerwiukk said[here](https://github.com/fishjam-cloud/web-client-sdk/pull/69#discussion_r1731092311 comment)